src/common/charset/Charset.h

```diff
@@ -86,6 +146,13 @@ class CharsetInfo final {
      * Description information of supported charsets
      */
     std::unordered_map<std::string, CharsetDesc> charsetDesc_;
+
+    /**
+     * Mapping collation to locale
+     */
+    std::unordered_map<std::string, std::string> collationToLocale_ = {
```

Comment:
We should support utf8_general_ci  at least.

Reply:
My test results show that the Chinese sorting results are the same in both utf8_bin and utf8_general_ci. utf8_general_ci is just a sort that ignores case.
I personally feel that there is no use case for utf8_general_ci?

Reply:
Fixed，We currently support two charset:
```
utf8
utf8mb4
```
We have extended utf8 to handle 4-byte characters, so
utf8 and utfmb4 are equivalent

Four collations are supported:
```
utf8_bin
utf8_general_ci
utf8mb4_bin
utf8mb4_general_c
```i